### ISSUE TITLE
Add project to awesome-go

### ISSUE THREAD

**Issue by stigok** [AllenDang/giu#37]
There's a list of [awesome Go projects called awesome-go][awesome-go]. None of the other GUI projects there seem to be similar to this one.

[awesome-go]: https://github.com/avelino/awesome-go#gui

**Comment by AllenDang** [comment]
@stigok Aha, thanks. I will make a pr today.

**Comment by AllenDang** [comment]
@stigok I need to write testing code to increase coverage before doing that. That may cost some time.

**Comment by stigok** [comment]
Yeah, I saw. But the need to have a coverage report may not necessarily mean that you need tests, but that the users can see how much/little of your codebase is covered. I found two alternatives that may be easy to set up
- https://github.com/marketplace/codacy
- https://github.com/marketplace/codecov

I don't mean to hassle you. Just close if you're not up for this work :)

**Comment by AllenDang** [comment]
@stigok All kinds of pr is welcome! I'm thinking about how to do the tests now. It seems kind of hard to cover C++ part...
I'm checking fyne, they seems did a very high coverage.

**Comment by jcbritobr** [comment]
This is the best gui project for go today. The really effective one. A very nice work.

**Comment by gucio321** [comment]
@AllenDang I think to make this happen tests are not essential - it is rather difficult to cover project with tests because it is a wrapper of C code which is tested on its own.
The thing is that we need to handle all the issues remaining open in the project since a longer time. I'm trying to organize them at the moment - it is many issues which are open but shouldn't in fact.
